exercises/node-javascript/Makefile

```diff
@@ -6,7 +6,8 @@ install_node:
 	@echo "node" > ~/.nvmrc
 	@curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
 	@NVM_DIR=/home/ec2-user/.nvm . "$$NVM_DIR/nvm.sh"; nvm install node
-	echo 'export AWS_SDK_LOAD_CONFIG=1' >> ~/.bashrc
+	availability_zone=$(curl http://169.254.169.254/latest/meta-data/placement/availability-zone) 
+	echo 'export AWS_REGION='${availability_zone%?} >> ~/.bashrc
```

Comment:
This looks weird, can we please cut an issue to track what's going on here and add a comment linking to it?

Reply:
#75 is to track getting to a complete solution.  This really should not be our issue to hack.